ui/app/pages/send/send-content/send-content.container.js

```diff
@@ -5,15 +5,17 @@ import {
   getSendTo,
 } from '../send.selectors'
 import {
-  getAddressBook,
+  getAddressBookEntry,
 } from '../../../selectors/selectors'
 import actions from '../../../store/actions'
 
 function mapStateToProps (state) {
+  const ownedAccounts = accountsWithSendEtherInfoSelector(state)
+  const to = getSendTo(state)
   return {
-    to: getSendTo(state),
-    addressBook: getAddressBook(state),
-    ownedAccounts: accountsWithSendEtherInfoSelector(state),
+    isOwnedAccount: !!ownedAccounts.find(({ address }) => address.toLowerCase() === to.toLowerCase()),
+    contact: getAddressBookEntry(state, to),
+    to,
```

Comment:
`to` can be removed here - it's no longer expected as a prop in the component.

Reply:
However, it is used in `mergeProps` within this file.

Reply:
Ah sorry, missed that 👍

Reply:
I'm still not certain that we strictly need it... we might simplify the code base without it